### PR TITLE
Unify logic to ChangeStringLiteral

### PR DESCRIPTION
**Pull request by JLLeitschuh**
Refactors some code to use the common `ChangeStringLiteral.withStringValue` method.

### CODE REVIEW

File: rewrite-gradle/src/main/java/org/openrewrite/gradle/ChangeDependencyArtifactId.java [hunk 1]
```diff
@@ -33,21 +34,21 @@
...
     String newArtifactId;
 
     @Option(displayName = "Dependency configuration",
-            description = "The dependency configuration to search for dependencies in.",
-            example = "api",
-            required = false)
+        description = "The dependency configuration to search for dependencies in.",
+        example = "api",
+        required = false)
     @Nullable
```
Comment (JLLeitschuh) [inline rewrite-gradle/src/main/java/org/openrewrite/gradle/ChangeDependencyArtifactId.java:52]: Can someone check the autoformat intellij settings into source control? That would help greatly.

Comment (jkschneider) [reply rewrite-gradle/src/main/java/org/openrewrite/gradle/ChangeDependencyArtifactId.java:52]: They are the default intellij settings, which we adhere to so they don't need to be committed.

Comment (JLLeitschuh) [reply rewrite-gradle/src/main/java/org/openrewrite/gradle/ChangeDependencyArtifactId.java:52]: Okay, so this is the default formatting, so this should theoretically be committed like this then

Comment (pway99) [reply rewrite-gradle/src/main/java/org/openrewrite/gradle/ChangeDependencyArtifactId.java:52]: can you check your Java tabs and indents?  I am going to guess your continuation is 4 and not 8.

**Comment by pway99** [comment]
Thanks @JLLeitschuh

**Pull request merged**